addons/udes_priorities/models/priority.py

```diff
@@ -26,6 +26,14 @@ class UdesPriority(models.Model):
         default=lambda self: uuid4(),
     )
 
+    @api.multi
+    @api.depends("reference")
+    def _get_default_affected_pickings(self):
+        Picking = self.env["stock.picking"]
+        self.affected_pickings = Picking.search(
+            [("priority", "=", self.reference), ("state", "not in", ("cancel", "done")), ]
```

Comment:
We are using the same domain in `_number_of_oustanding_pickings` and they should be aligned. Would be good to have a function that generates this domain and use it in both methods.

Reply:
We could also call this method for the domain of the picking view if we do the change suggested in: https://github.com/unipartdigital/udes-open/pull/522#discussion_r741020847

Reply:
I think to remove at all _number_of_oustanding_pickings method, and to change with the field affected_pickings.
When first was done , it was used i think search_count for performance but now that we are adding affected_pickings we dont need at all _number_of_oustanding_pickings. Even if the number is needed can use length but as far as i saw is not needed at the moment, just True or False we need

Reply:
I totally agree :)